Build/Blazorise.props

```diff
@@ -1,64 +1,152 @@
-		<DefineConstants>$(DefineConstants);NET6_0</DefineConstants>
-	</PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'net7.0' ">
+    <DefineConstants>$(DefineConstants);NET7_0</DefineConstants>
+  </PropertyGroup>
 
-	<PropertyGroup Condition=" '$(TargetFramework)' == 'net7.0' ">
-		<DefineConstants>$(DefineConstants);NET7_0</DefineConstants>
-	</PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'net8.0' ">
+    <DefineConstants>$(DefineConstants);NET8_0</DefineConstants>
+  </PropertyGroup>
 
-	<PropertyGroup Condition=" '$(TargetFramework)' == 'net8.0' ">
-		<DefineConstants>$(DefineConstants);NET8_0</DefineConstants>
-	</PropertyGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
+    <PackageReference Include="Microsoft.AspNetCore.Components" Version="6.0.*" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="6.0.*" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.*" />
+  </ItemGroup>
 
-	<ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
-		<PackageReference Include="Microsoft.AspNetCore.Components" Version="6.0.*" />
-		<PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="6.0.*" />
-		<PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.*" />
-	</ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net7.0' ">
+    <PackageReference Include="Microsoft.AspNetCore.Components" Version="7.0.*" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="7.0.*" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="7.0.*" />
+  </ItemGroup>
 
-	<ItemGroup Condition=" '$(TargetFramework)' == 'net7.0' ">
-		<PackageReference Include="Microsoft.AspNetCore.Components" Version="7.0.*" />
-		<PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="7.0.*" />
-		<PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="7.0.*" />
-	</ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net8.0' ">
+    <PackageReference Include="Microsoft.AspNetCore.Components" Version="8.0.*" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="8.0.*" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="8.0.*" />
+  </ItemGroup>
+
+  <UsingTask
+	  TaskName="ReplaceVersionInJsFiles"
+	  TaskFactory="RoslynCodeTaskFactory"
+	  AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.Core.dll">
+    <ParameterGroup>
+      <SourceFiles ParameterType="Microsoft.Build.Framework.ITaskItem[]" Required="true" />
+      <Version ParameterType="System.String" Required="true" />
+    </ParameterGroup>
+    <Task>
+      <Using Namespace="System"/>
+      <Using Namespace="System.IO"/>
+      <Using Namespace="System.Text.RegularExpressions" />
+      <Code Type="Fragment" Language="cs">
```

Comment:
Do you think it makes sense to create a console app that do the same as this and we execute it instead of having the code build into the project configuration? We can keep the `JsTrim.exe`(of however we call it) in the NuGet subfolder.

Reply:
No. This is much more elegant in my opinion. It's an integral part of the build process and makes sure the versions are always correct, does not depend on human error.
Also this is used to publish master 1.5.0 weekly correctly version wise.

Reply:
OK, we can always revisit it later.